### DISCUSSION TITLE
duckdb dialect

### DISCUSSION THREAD

**Discussion by eitsupi** [PRQL/prql#1552] [Q&A] [answered]
While working on [updating prqlr](https://github.com/eitsupi/prqlr/pull/35), I noticed that duckdb dialect has been implemented. (Fantastic exclude support!)
However, this is not in the list of dialects available in prql-js and does not seem to be mentioned in the book.

https://github.com/PRQL/prql/blob/ad543b80426981e0d2300d38061633226d3ea9dc/prql-js/src/lib.rs#L110-L119
https://github.com/PRQL/prql/blob/ad543b80426981e0d2300d38061633226d3ea9dc/book/src/queries/header.md?plain=1#L34-L43

From what I have played with R this seems to work well, but I was wondering if it is likely that that is not being intentionally listed.

**Comment by max-sixty** [comment, marked as answer]
Thanks for the question @eitsupi . We should update that. Feel free to transfer this to an issue to track that.

Contributions welcome, would be a reasonable first PR for anyone who wanted to get involved.

**Reply by eitsupi** [reply]
Done #1559.
Thanks!
